docs/admin.rst

```diff
@@ -250,52 +250,36 @@ smartphone authenticator app or a Yubikey for two-factor authentication. If an
 app is to be used, the journalist should install it before proceeding with the
 account setup.
 
-  .. include:: includes/otp-app.txt
+.. include:: includes/otp-app.txt
 
-* First, click **Admin** in the top right corner of the page to load the *Admin Interface*:
+#. Click **Admin** in the top right corner of the page to load the *Admin Interface*.
 
   |SecureDrop admin home|
 
-*  Once there, click **Add User** to add a new user:
+#. Click **Add User** to add a new user.
```

Comment:
The way it's currently formatted, this will restart the numbering at 1.

Reply:
Good catch. I'll fix that, I think the blank lines may be at fault but I'll figure that out.

Reply:
Fixed. It wasn't the blank lines, but the indentation of the image references. Four spaces (not 3) are needed for the image to be considered part of the list item. If the images aren't considered as part of the list items, a new list starts after each image.